homeassistant/components/home_connect/__init__.py

```diff
@@ -486,6 +486,15 @@ async def async_service_set_program_and_options(call: ServiceCall):
                 )
             elif option in PROGRAM_OPTIONS:
                 option_key = PROGRAM_OPTIONS[option][0]
+                if (
+                    option_key
+                    is OptionKey.CONSUMER_PRODUCTS_COFFEE_MAKER_COFFEE_MILK_RATIO
+                ):
+                    if value in (67, 68):
+                        value = 67
+                    else:
+                        value = round(value / 5) * 5
```

Comment:
Where is the `value` defined? What can it be?

According to the docs there's no option for 15, 35, 45, between 10 and 90.

https://api-docs.home-connect.com/programs-and-options/#coffee-machine_coffee-milk-ratio-option

Reply:
True, maybe we should switch to a select selector as I stated in the [above comment](https://github.com/home-assistant/core/pull/138593#issuecomment-2660879042) to simplify this?

Reply:
Yeah, all the options that have values that are enumeration types according to the docs should be select selector. We shouldn't try to be smart.